### ISSUE TITLE
Memory pressure while downloading a large size file using GoldRaccoon

### ISSUE THREAD

**Issue by htaiwan** [albertodebortoli/GoldRaccoon#30]
I use GoldRaccoon as my 3rd ftp library. When I download large size (> 500MB) file, my app will crash in memory pressure. The snapshot as below that using instrument to detect the memory allocations in my app. How do I handle these allocation when getting low memory warning ??

![screen shot 2015-06-29 at 10 14 19 am](https://cloud.githubusercontent.com/assets/1651917/8400595/d2d470e4-1e53-11e5-95d6-b47a195d02c5.png)

**Comment by ValentinKuznetsov** [comment]
Agree, this happens because the downloaded data is not written to the disk chunk by chunk, but stored in the memory instead.
